### PR TITLE
New package: tilix-1.8.5

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/tilix/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'tilix'
+pkgname=tilix
+version=1.8.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel glib-devel gtk+3-devel ldc po4a pkg-config"
+makedepends="dconf-devel gtkd-devel libglib-devel libvted-devel libX11-devel"
+short_desc="tiling terminal emulator for Linux"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+homepage="https://gnunn1.github.io/tilix-web/"
```
Comment (maxice8) [inline srcpkgs/tilix/template]: homepage= should be directly above distfiles=

File: srcpkgs/tilix/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'tilix'
+pkgname=tilix
+version=1.8.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel glib-devel gtk+3-devel ldc po4a pkg-config"
```
Comment (maxice8) [inline srcpkgs/tilix/template]: is gtk+3-devel really required on hostmakedepends ?

**Comment by maxice8** [comment]
Please rebase against master as https://github.com/void-linux/void-packages/commit/06133523007575b43e7ab481d34313a233f1b31c was merged

**Comment by Cogitri** [comment]
Done

**Comment by Cogitri** [comment]
I suppose the
```
case "$XBPS_TARGET_MACHINE" in
	*-musl) broken="ldc is broken" ;;
esac
```

stuff should be removed from gtkd because ldc works now?

**Comment by maxice8** [comment]
On Wed, Sep 05, 2018 at 06:43:24PM +0000, Rasmus Thomsen wrote:
> I suppose the
> ```
> case "$XBPS_TARGET_MACHINE" in
> 	*-musl) broken="ldc is broken" ;;
> esac
> ```
> 
> stuff should be removed from gtkd because ldc works now?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2323#issuecomment-418837895

Yes

**Pull request merged**